lib/sls-api-file-delete-error.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+class SlsApiFilDeleteError extends Error {
+
+	static get messages() {
+
+		return {
+			BUCKET_NOT_DEFINED: 'Bucket not defined',
+			BUCKET_NOT_STRING: 'Bucket should be return a string',
+			ENTITY_ID_FIELD_NOT_DEFINED: 'EntityIdField not defined',
+			ENTITY_ID_FIELD_NOT_STRING: 'EntityIdField should be return a string',
+			MODEL_NOT_DEFINED: 'Model not defined',
+			MODEL_IS_NOT_MODEL_CLASS: 'Model is not a Model Class',
+			FILE_RECORD_NOT_FOUND: 'file record not found'
+		};
+
+	}
+
+	constructor(err) {
+		super(err);
+		this.message = err.message || err;
+		this.name = 'SlsApiFilDeleteError';
```

Comment:
```suggestion
		this.name = 'SlsApiFilDeleteError';
		
		if(err instanceof Error)
			this.previousError = err;
```

Y esto en todos los errores. Así estandarizamos los errores entre todos los :package: